### PR TITLE
SSL Error handling, retry

### PR DESCRIPTION
**Pull request by MohammadMQ**

### CODE REVIEW

File: library/src/main/java/com/queue_it/androidsdk/QueueActivity.java [hunk 1]
```diff
@@ -126,24 +126,16 @@ public void onReceivedError(WebView view, WebResourceRequest request, WebResourc
 
             @Override
             public void onReceivedSslError(WebView view, final SslErrorHandler handler, SslError error) {
+                handler.cancel();
+                broadcastQueueActivityFailed("SslError, code: " + error.getPrimaryError()); // <- ny metode, der skal implementeres
```
Comment (frwiqueueit) [inline library/src/main/java/com/queue_it/androidsdk/QueueActivity.java]: remove code comment

File: library/src/main/java/com/queue_it/androidsdk/QueueITEngine.java [hunk 2]
```diff
@@ -163,6 +169,14 @@ public void onReceive(Context context, Intent intent) {
         }
     };
 
+    private BroadcastReceiver _queueSslErrorBroadcastReceiver = new BroadcastReceiver() {
```
Comment (frwiqueueit) [inline library/src/main/java/com/queue_it/androidsdk/QueueITEngine.java]: rename to _queueErrorBroadcastReceiver

File: library/src/main/java/com/queue_it/androidsdk/QueueActivity.java [hunk 3]
```diff
@@ -126,24 +126,16 @@ public void onReceivedError(WebView view, WebResourceRequest request, WebResourc
...
-                builder.setNegativeButton(android.R.string.cancel, new DialogInterface.OnClickListener() {
-                    @Override
-                    public void onClick(DialogInterface dialog, int which) {
-                        handler.cancel();
-                    }
-                });
-
-                final AlertDialog dialog = builder.create();
-                dialog.show();
+            private void broadcastQueueActivityFailed(String sslErrorMessage)
```
Comment (frwiqueueit) [inline library/src/main/java/com/queue_it/androidsdk/QueueActivity.java]: renname sslErrorMessage to errorMessage

File: library/src/main/java/com/queue_it/androidsdk/QueueActivity.java [hunk 4]
```diff
@@ -126,24 +126,16 @@ public void onReceivedError(WebView view, WebResourceRequest request, WebResourc
...
+            {
+                Intent intent = new Intent("queue-activity-failed-ssl-error");
```
Comment (frwiqueueit) [inline library/src/main/java/com/queue_it/androidsdk/QueueActivity.java]: rename to on-queue-error

File: library/src/main/java/com/queue_it/androidsdk/QueueActivity.java [hunk 5]
```diff
@@ -126,24 +126,16 @@ public void onReceivedError(WebView view, WebResourceRequest request, WebResourc
...
+                intent.putExtra("ssl-error-message", sslErrorMessage);
```
Comment (frwiqueueit) [inline library/src/main/java/com/queue_it/androidsdk/QueueActivity.java]: rename to 'error-message' and errorMessage

File: library/src/main/java/com/queue_it/androidsdk/QueueITEngine.java [hunk 6]
```diff
@@ -7,11 +7,13 @@
 import android.content.IntentFilter;
 import android.net.ConnectivityManager;
 import android.net.NetworkInfo;
+import android.net.http.SslError;
```
Comment (frwiqueueit) [inline library/src/main/java/com/queue_it/androidsdk/QueueITEngine.java]: remove not needed imports

File: library/src/main/java/com/queue_it/androidsdk/QueueActivity.java (hunk 1 above)
Comment (frwiqueueit) [inline library/src/main/java/com/queue_it/androidsdk/QueueActivity.java]: rename to broadcastQueueError

File: library/src/main/java/com/queue_it/androidsdk/QueueActivity.java (hunk 3 above)
Comment (frwiqueueit) [inline library/src/main/java/com/queue_it/androidsdk/QueueActivity.java]: this entire method should be moved to same location as the rest of the broadcast methods.

Comment (MohammadMQ) [reply library/src/main/java/com/queue_it/androidsdk/QueueActivity.java]: You want this method to be moved to QueueITEngine.java file?

**Pull request merged**